includes/class-wporg-gp-translation-events-route.php

```diff
@@ -91,23 +91,66 @@ public function events_edit( $event_id ) {
 	 * @return void
 	 */
 	public function events_details( $event_slug ) {
-		if ( ! is_user_logged_in() ) {
+		$user = wp_get_current_user();
+		if ( ! $user ) {
 			$this->die_with_404();
 		}
 		$event = get_page_by_path( $event_slug, OBJECT, 'event' );
 
 		if ( ! $event ) {
 			$this->die_with_404();
 		}
+		$event_id           = $event->ID;
 		$event_title        = $event->post_title;
 		$event_description  = $event->post_content;
 		$event_start_date   = get_post_meta( $event->ID, '_event_start_date', true ) ?? '';
 		$event_end_date     = get_post_meta( $event->ID, '_event_end_date', true ) ?? '';
 		$event_locale       = get_post_meta( $event->ID, '_event_locale', true ) ?? '';
 		$event_project_name = get_post_meta( $event->ID, '_event_project_name', true ) ?? '';
+
+		$attending_event_ids = get_user_meta( $user->ID, self::USER_META_KEY_ATTENDING, true ) ?? [];
```

Comment:
We need to use `?:` since get_user_meta doesn't return `null` in the case of not found but something falsy. Likely we need to change the other lines above this one too.
```suggestion
		$attending_event_ids = get_user_meta( $user->ID, self::USER_META_KEY_ATTENDING, true ) ?: [];
```

I stumbled on this because I received this error:  Uncaught TypeError: array_key_exists(): Argument #2 ($array) must be of type array, string given in includes/class-wporg-gp-translation-events-route.php:112

Reply:
Ooops, good catch. Fixed all of them.